libbeat/docs/release-notes/5.0.0.asciidoc

```diff
@@ -0,0 +1,263 @@
+[[release-notes-5.0.0]]
+=== 5.0.0 Combined Release Notes
+
+The list below covers changes between 1.3.1 to 5.0.0-GA.
```

Comment:
If users want the list of changes for 1.3. I don't think these relnotes belong in the 5.0 docs.